packages/neuron-wallet/src/models/chain/transaction-point.ts

```diff
@@ -0,0 +1,23 @@
+import TypeChecker from "utils/type-checker"
+
+export default class TransactionPoint {
+  constructor(
+    public blockNumber: string,
+    public txHash: string,
+    public index: string
+  ) {
+    this.blockNumber = BigInt(blockNumber).toString()
```

Comment:
`blockNumber` and `index` are modified when assigned, thus constructor assignment shorthand is not shorthand anymore. How about declaring them as public instance properties explicitly?

Reply:
OK，you means 
```
public blockNumber: string

constructor(blockNumber: string)
```
 ?

Reply:
All shorthand assignments are deleted